index.html

```diff
@@ -1,115 +1,276 @@
+				z-index: -1;
+				min-width: 96px;
+				min-height: 384px;
+				position: absolute;
+				background-size: 96px auto;
+				background-image: url("../images/syndicate.png");
+				animation: syndicate2Move 55s linear infinite;
+			}
+			
+			html {
+				font-size: 14px;
+			}
+			
+			main {
+				margin-left: 12.5%;
+				margin-right: 12.5%;
+				margin-top: 2rem;
+				min-width: 12rem;
+				font-family: 'Lato', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+			}
+			
+			#banner {
+				white-space: nowrap;
+				font-size: 7rem;
+				font-weight: 700;
+				color: #fff;
+				background-image: url("../images/logo.png");
+				background-repeat: no-repeat;
+				background-size: 120px;
+				padding-left: 120px;
+				padding-bottom: 1rem;
+			}
+			
+			nav {
+				display: flex;
+				overflow: hidden;
+				background-color: #fff;
+				border-radius: 0.3rem;
+				margin-bottom: 1rem;
+			}
+			
+			a {
+				text-decoration: none;
+			}
+			
+			nav a {
+				padding: 1rem;
+				flex: 1 1 100%;
+				text-align: center;
+				vertical-align: middle;
+				transition: background-color 0.25s ease-in;
+				border-left: thin solid #ddd;
+				color: #000;
+			}
+			
+			nav a:hover {
+				background-color: #eee;
+			}
+			
+			#content {
+				background-color: #fff;
+				border-radius: 0.3em;
+				padding: 1rem;
+			}
+			
+			#statusDisplay {
+				display: flex;
+			}
+			
+			#statusDisplay div {
+				flex: 1 1 100%;
+				text-align: center;
+			}
+			
+			footer {
+				text-align: center;
+				padding: 2rem;
+				font-size: 0.75rem;
+				color: #aaa;
+			}
+			
+			footer a {
+				color: #fff;
+			}
+			
+			#content p {
+				line-height: 1.5rem;
+			}
+			
+			#players {
+				font-size: 5rem;
+				color: #2185d0;
+			}
+			
+			#mode {
+				font-size: 5rem;
+				color: #00b5ad;
+			}
+			
+			#duration {
+				font-size: 5rem;
+				color: #6435c9;
+			}
+			
+			div header {
+				font-weight: 700;
+			}
+			
+			a img {
+				width: 2rem;
+				height: 2rem;
+				display: inline;
+			}
+			
+			#forums, #wiki, #github, #play {
+				width: 1em;
+				height: 1em;
+				display: inline-block;
+				background-repeat: no-repeat;
+				background-position: center;
+				padding: 1em 1.5em 1em 1em;
+				margin: -1em;
+			}
+			
+			#forums {
+				background-image: url("../images/comment.svg");
+				background-size: 1.5em;
+			}
+			
+			#wiki {
+				background-image: url("../images/sticky-note.svg");
+				background-size: 1.2em;
+			}
+			
+			#github {
+				background-image: url("../images/github.svg");
+				background-size: 1.4em;
+			}
+			
+			#play {
+				background-image: url("../images/gamepad.svg");
+				background-size: 1.6em;
+			}
+		</style>
+	</head>
+	<body>
+		<div id="monkey"></div>
+		<div id="meteor1"></div>
+		<div id="meteor2"></div>
+		<div id="syndicate1"></div>
+		<div id="syndicate2"></div>
+		<main>
+			<header id="banner">Baystation 12</header>
+			<nav>
+				<a href="//baystation12.net/forums"><span id="forums"></span> Forums</a>
```

Comment:
Use `<img id="forums" />` etc. (okay to omit `alt` for this purpose...but alt does actually function the same even though the image is defined in CSS)